src/lib/y2storage/blk_device.rb

```diff
@@ -791,7 +791,7 @@ def encrypt_with_method(method, dm_name, **method_args)
     #
     # @return [Filesystems::MountByType]
     def preferred_mount_by
-      Filesystems::MountByType.best_for(self, possible_mount_bys)
+      Filesystems::MountByType.best_for(self, possible_mount_bys & Filesystems::MountByType.all)
```

Comment:
I find this inconsistent with the way it's done for `MountPoint`.  In `MountPoint` you redefined `#possible_mount_bys` to return a filtered list. Here, `#possible_mount_bys` still returns an unfiltered list and the filter is done at `preferred_mount_by`.

Reply:
well, reason here is that it is private method and this is its only usage.